dynamic_instrumentation/src/source.rs

```diff
@@ -0,0 +1,67 @@
+use c2rust_analysis_rt::mir_loc::MirPlace;
+use rustc_middle::mir::{Local, Operand, Place, Rvalue};
+
+use crate::util::Convert;
+
+pub trait Source {
```

Comment:
i think this module should also be folded into `point`, for the same reasons `into_operand` should